gui/qt/amountedit.py

```diff
@@ -104,3 +104,18 @@ def setAmount(self, amount):
 class BTCkBEdit(BTCAmountEdit):
     def _base_unit(self):
         return BTCAmountEdit._base_unit(self) + '/kB'
+
+class BTCkBEdit2(BTCAmountEdit):
+    def _base_unit(self):
+        return 'sats' + '/kB'
+    def get_amount(self):
+        try:
```

Comment:
Only catch the exceptions you are expecting to catch, with explicit exception classes, otherwise no-one knows what you were intending to intentionally catch and anything unexpected is hidden and shows up as a side-effect elsewhere.

Reply:
Also, do you know that self.text() will ever not be a str?  I suspect the str() call is pointless.

Reply:
Good points; I am simply overriding inherited functions with much of the same code.